### ISSUE TITLE
[visualforce] False positive on VfUnescapeEl with new Message Channel feature

### ISSUE THREAD

**Issue by pchittum** [pmd/pmd#2620]
**Affects PMD Version:** 
6.25.0

**Rule:**
[VfUnescapeEl](https://pmd.github.io/latest/pmd_rules_vf_security.html#vfunescapeel)

**Description:**
There's a new feature called Lightning Message Service that is surfaced through a global variable `$MessageChannel`. The only way to access it is to bind to it in Visualforce markup. 

This produces a false positive when PMD encounters this, as there's no reason to escape this content. It could never be entered by a user. 

Docs: https://developer.salesforce.com/docs/atlas.en-us.222.0.pages.meta/pages/pages_variables_global_messagechannel.htm and https://developer.salesforce.com/docs/atlas.en-us.222.0.pages.meta/pages/message_channel_intro.htm

**Code Sample demonstrating the issue:**

```html
    <script type="text/javascript">
        // Binding message channel to variable accessible to static resource.
        window.util = {
            messageChannel: '{!$MessageChannel.Record_Selected__c}'
        };
    </script>
```

**Expected outcome:**

This usage should produce no rule violations with PMD. I believe it should be included in the list of safe global variable found here: https://github.com/pmd/pmd/blob/596f27bf93f0af1b7220226fec40970696e6ba09/pmd-visualforce/src/main/java/net/sourceforge/pmd/lang/vf/rule/security/VfUnescapeElRule.java#L290-L298

**Running PMD through:** *[CLI | Ant | Maven | Gradle | Designer | Other]*
Other

**Comment by adangel** [comment]
Was reverted, so reopen.
